.mailmap

```diff
@@ -0,0 +1,41 @@
+Aaron Hopkinson <aaron.hopkinson@metoffice.gov.uk> <arh89@users.noreply.github.com>
+Anna Booton <anna.booton@metoffice.gov.uk> <anna.booton@metoffice.gov.uk>
+Anja Schubert <anja.schubert@bom.gov.au> <57921950+anja-bom@users.noreply.github.com>
+Belinda Trotta <73675905+btrotta-bom@users.noreply.github.com> <73675905+btrotta-bom@users.noreply.github.com>
+Benjamin Ayliffe <benjamin.ayliffe@metoffice.gov.uk> <benjamin.ayliffe@metoffice.gov.uk>
+Benjamin Owen <benjamin.owen@bom.gov.au> <71359048+benowen-bom@users.noreply.github.com>
+Ben Fitzpatrick <ben.fitzpatrick@metoffice.gov.uk> <ben@seven.powderham>
+Bruno P. Kinoshita <kinow@users.noreply.github.com> <kinow@users.noreply.github.com>
+Caroline Jones <caroline.jones@metoffice.gov.uk> <caroline.jones@metoffice.gov.uk>
+Caroline Jones <caroline.jones@metoffice.gov.uk> <caroline.jones@metoffice.gov.uk>
```

Comment:
Both Caroline Jones lines are identical.

Reply:
Not no more! 👍